elastic/kibana
# [ML] Decouple data_visualizer from MapEmbeddable

Part of https://github.com/elastic/kibana/issues/182020

### test instructions
1. install web logs sample data
2. Open discover
3. In table, click "Field statistics"
4. Expand `geo.dest` row. Verify choropleth map is displayed.
5. Expand `geo.coordinates` row. Verify vector map is displayed.
